tools/DevInsights/DevHome.DevInsights/Strings/en-us/Resources.resw

```diff
@@ -1199,15 +1199,47 @@ Dump File creation time: {1}</value>
+    <value>Regular Expression for WER crash dump bucket</value>
     <comment>Label for a textbox that will contain the regular expression a tool uses to determine the bucketing string of a cab.</comment>
   </data>
+  <data name="ProcessListHeaderTextBlock.ToolTipService.ToolTip" xml:space="preserve">
+    <value>A list of all running processes. You can select an item from the list to target. You can also type a name in the search box to find that process. The drop-down contains a list of all items that are filtered out by default - you can select any of these to include them in the main list. Another way to select your target process is to bring its window to the foreground and then use the hotkey Windows+F12.</value>
+    <comment>{Locked="Windows+F12"} Description of the Process list page.</comment>
+  </data>
+  <data name="WERHeaderTextBlock.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Recent crash reports available on this machine. By default, this is filtered to your target app (if any). You can select any dump file in the list to open it in a dump analyzer tool of your choice. If you want to change your choice of dump analyzer tool, go to Windows Settings &gt; Apps &gt; Default apps &gt; Set default for ".dmp" file type.</value>
+    <comment>{Locked=".dmp"} Description of the Crash reports page.</comment>
+  </data>
+  <data name="InsightsHeaderTextBlock.ToolTipService.ToolTip" xml:space="preserve">
+    <value>As Dev Insights gathers context data for your target app, it can analyze the data, and in some cases it can provide insights into the nature of the problem. If Dev Insights does identify insights, they will be listed here. (Note that this is a work in progress, and initially there are very few cases where Dev Insights can provide definitive analysis.)</value>
+    <comment>Description of the Insights page.</comment>
+  </data>
+  <data name="ResourceUsageHeaderTextBlock.ToolTipService.ToolTip" xml:space="preserve">
+    <value>A list of a wide range of resource usage and sensor items for both your target app (if any), and for the system as a whole.</value>
+    <comment>Description of the Resource usage page.</comment>
+  </data>
+  <data name="ModulesHeaderTextBlock.ToolTipService.ToolTip" xml:space="preserve">
+    <value>All the modules (DLLs, etc) that have been loaded by your target process.</value>
+    <comment>Description of the Loaded modules page.</comment>
+  </data>
+  <data name="WinLogsHeaderTextBlock.ToolTipService.ToolTip" xml:space="preserve">
+    <value>As you exercise the features of your target app, this will generate log outputs in various forms. By default, this page tracks Event Tracing for Windows (ETW) logs, the system Event log, and Windows Error Reporting (WER) logs. You can also enable Debug Output tracking, althought this can produce a large amount of output, depending on the app. Dev Insights analyzes log output and in some cases can generate insights.</value>

```

Comment:
ETW logs are enabled only when the user is a member of Performance Log Users group, otherwise it is disabled. Although, I am not sure, if we want to add implementation detail here.